### DISCUSSION TITLE
Environment variable in config.yaml

### DISCUSSION THREAD

**Discussion by Bloopps** [Eugeny/tabby#5837] [Q&A]
Hi,
I'm looking for a way to use an environment variable '$USER' instead of the username in the config.yaml file
Is there a particular syntax?

**Comment by Eugeny** [comment]
No, there's no way to substitute variables in the config

**Comment by Bloopps** [comment]
Would it be possible to add this feature?

I use terminus-cloud-settings-sync to sync our config with my colleagues and it would be really helpful, I think this feature could be useful for other people.

Anyway, thanks for your work on this great tool :+1:

**Reply by Eugeny** [reply]
Feel free to open an issue for it, but I'm not sure I'll be able to find time for it soon :( - PRs are welcome as always
